tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_002_pos.ksh

```diff
@@ -50,14 +50,14 @@ function cleanup
 	for pool in $TESTPOOL $TESTPOOL1 $TESTPOOL2 $TESTPOOL3 $TESTPOOL4 \
 		$TESTPOOL5 $TESTPOOL6
 	do
-		poolexists $pool && destroy_pool $pool
+		poolexists $pool && log_must destroy_pool $pool
```

Comment:
You can actually get rid of the `poolexists` check. `destroy_pool` already performs the check. You also don’t want to use `log_must`, if one destroy fails, you’d still want to attempt the other ones.